test/models/test_session_traceable.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class TestTraceable < ActiveSupport::TestCase
```

Comment:
```suggestion
class TestSessionTraceable < ActiveSupport::TestCase
```

Reply:
Same reason with `TestLimitable`, it will cause a superclass mismatch with integration test.

Reply:
Mostly the name of the class is a bit of an issue because it doesn't match the name of the file.  I'd also like it to have a name that is related to the feature you are implementing.